gapis/api/templates/api.proto.tmpl

```diff
@@ -28,7 +28,7 @@
   {{end}}
 
   {{range $c := $.Classes}}
-    {{if (GetAnnotation $c "serialize")}}{{Template "ClassEntry" $c}}{{end}}
+    {{if not (GetAnnotation $c "noserialize")}}{{Template "ClassEntry" $c}}{{end}}
```

Comment:
If you've killed everything that responds to the `@serialize` annotation, please can you remove the uses of the annotation?